kiva/_cython_speedups.pyx

```diff
@@ -1,5 +1,5 @@
-from libc.stdint cimport uint8_t
 import numpy as np
+cimport numpy as np
```

Comment:
Style nit: I may be alone in this (certainly the Cython numpy tutorials do it like you do), but I don't like `import numpy as np` and `cimport numpy as np`. I usually `cimport numpy as cnp` to distinguish the later references.

Or in this case, since you are importing only one thing: `from numpy cimport uint8_t`

Either way, LGTM.